src/operand/reference.ts

```diff
@@ -86,11 +87,27 @@ function contextValueLookup(ctx: Context, key: string): Result {
   return undefined
 }
 
+/**
+ * Converts a value to a specified data type, returns original value if not parseable.
+ * @param value value to parse as data type
+ */
+function toDataType(value: Result, dataType: string | undefined): Result {
+  switch (dataType) {
+    case 'Number':
+      return toNumber(value)
+    case 'String':
+      return toString(value)
```

Comment:
I guess, also for Number and String, we may console.warn as well, if evaluated as `undefined` - really to give us the change to fix these, as we define the conditions.

Reply:
b8f7162